hotel-management/README.md

```diff
@@ -0,0 +1,105 @@
+
+## TIMELINE
+
+- Expected: 70 hours  (from 2024/03/27 - 2024/04/10)
+
+## TEAM SIZE
+
+- One developer.
+
+## TARGET
+
+- Understand and apply knowledge of HTML5, CSS3, JavaScript (with ES6 syntax).
+- Understand and apply TypeScript in React.
+- Get familiar with design React components.
+- Understand asynchronous programming and how to work with API.
+- Understanding how React hooks work.
+- Learn to debug with React developer tools.
+
+## TECHNICAL STACK
+
+- [HTML5](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5) - HTML5 is the latest evolution of the standard that defines HTML
+- [CSS3](https://developer.mozilla.org/en-US/docs/Web/CSS) - Cascading Style Sheets (CSS) is a stylesheet language used to describe the presentation of a document written in HTML.
+- [React 18](https://react.dev) - React is a JavaScript-based UI development library. Although React is a library rather than a language, it is widely used in web development.
+- [React-router-dom](https://reactrouter.com/en/main) - React Router is a customizable and flexible routing library for React, enabling navigation among views of various components.
+- [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript) - JavaScript is a lightweight interpreted programming language with first-class functions.
+- [TypeScript](https://www.typescriptlang.org) - TypeScript is an object-oriented programming language developed by Microsoft Corporation, and it is a subset of JavaScript.
+- [Eslint](https://eslint.org) - ESLint is a static code analysis tool for identifying problematic patterns found in JavaScript code.
+- [JSON Server](https://github.com/typicode/json-server) - Full fake REST API with zero coding in less than 30 seconds.
+- [Vite](https://vitejs.dev) - Vite is a build tool that aims to provide a faster and leaner development experience for modern web projects.
+
+## DESIGN
+
+- Design: [Home Page](https://www.figma.com/file/fktqRevXEV7CUhux1T5h0h/Hotel-Website-Wireframe-Design-(Community)-(Copy)?type=design&node-id=2%3A1030&mode=design&t=CvKNLXnmqRK4ois8-1), [Details Page](https://www.figma.com/file/fktqRevXEV7CUhux1T5h0h/Hotel-Website-Wireframe-Design-(Community)-(Copy)?type=design&node-id=2%3A765&mode=design&t=atNGPZ94tRDvQpNS-1).
+- Design Add-product page based on my own idea.
+
+## TASK MANAGEMENT
+
+- [Trello](https://trello.com/b/py0VHl0r/tiennguyen-internship-react)
+
+## REQUIREMENTS
+
+- [My Requirements](https://docs.google.com/document/d/1q2POcQAJleoKG9nD43pJci07Y4QRUmlfrqZxCqXMv2Y/edit#heading=h.1t3h5sft)
+
+## GETTING STARTED
+
+- Clone the repository
+
+```bash
+git clone https://github.com/TienNguyenDinh/training-react.git
+```
+
+- Move to the cloned repository folder
+
+```bash
+cd training-react
+```
+
+- Checkout to branch `react-practice`
+
+```bash
+git checkout react-practice
+```
+
+- Move to folder `hotel-management`
+
+```bash
+cd hotel-management
+```
+
+- Install packages
+
+```bash
+npm install
```

Comment:
Please do me a favor, use `pnpm` instead `npm`, update filelock then and requirement for pnpm, node